utility/utility_test.bzl

```diff
@@ -0,0 +1,18 @@
+load("//:oppia_android_test.bzl", "oppia_android_test")
+
+def utility_test(name, src, test_class):
```

Comment:
Similar to my comments on your other PR, I suggest simplifying this macro.

Reply:
done!